test/Sentry.Tests/GlobalSessionManagerTests.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Sentry.Testing;
+using Xunit;
+
+namespace Sentry.Tests
+{
+    public class GlobalSessionManagerTests
+    {
+        [Fact]
+        public void StartSession_ReleaseSet_CreatesNewSession()
```

Comment:
Where is release being set? Isn't clear by reading the code

Reply:
It pulls it from assembly version. I'll make it explicit.